frameworks/g-cloud-9/questions/services/schedulingBackups.yml

```diff
@@ -1,20 +1,17 @@
-name: "Scheduling backups"
-question: "How do users schedule backups?"
-optional: true
+name: Scheduling backups
+question: How do users schedule backups?
+
 depends:
-  -
-    "on": lot
+  - "on": lot
     being:
       - cloud-hosting
 
-type: checkboxes
+type: radios
 options:
-    -
-        label: "Users schedule backups through a web interface"
-        filter_label: "Web interface"
-    -
-        label: "Users contact the support team to schedule backups"
-        filter_label: "Support team"
-    -
-        label: "Supplier controls the whole backup schedule"
-        filter_label: "Supplier-controlled"
+  - label: Users schedule backups through a web interface
```

Comment:
They're generally ordered from 'best'/'most preferable' to 'worst'/'least preferable'; if there's no clear way to order them like this, it falls back to alphabetical.